### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by robb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +23,15 @@ It's a good idea to give visitors an easy way to send such requests; for example
...
-Optionally, a GET request to the same URL could respond with the emoji reaction counts.
+Optionally, a GET request to the same URL may respond with the emoji reaction counts.
+
+The response should be a JSON object mapping Emoji (as Strings) to their count (as Strings):
+
+```json
+{"❤️":"1","🫀":"2","❤️‍🔥":"3"}
+```
+
+If reaction counts are read-only, the server should respond with a 404.
```
Comment (robb) [inline README.md:34]: Is there a better status code?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
+The Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
 
 ## How
 
-Set up an endpoint to receive an Open Heart POST request like this one:
+Set up an endpoint to receive an Open Heart `POST` request like this one:
 
 ```
 curl -d '🫠'  -X POST '<url>'
```
Comment (robb) [inline README.md:10]: Would be nice if this example worked.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
 # Open Heart Protocol
 
-Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
+The Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
```
Comment (robb) [inline README.md:3]:
```suggestion
The Open Heart protocol lets an anonymous user sends an emoji reaction to a URL.
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
 
 ## How
 
-Set up an endpoint to receive an Open Heart POST request like this one:
+Set up an endpoint to receive an Open Heart `POST` request like this one:
 
 ```
 curl -d '🫠'  -X POST '<url>'
 ```
```
Comment (robb) [inline README.md:12]: Does this make sense?

```suggestion

A Open Heart message should contain of a single emoji sequence. However, the emoji sequence may be followed by arbitrary data which the server is expected to ignore. 

This allows HTML `<form>`s to post reactions using the Open Heart protocol through an empty input. In this case, the payload will be `🫠=`.
```

Comment (muan) [reply README.md:12]: Yup, I'll include a form markup example here as well.

File: README.md [hunk 5]
```diff
@@ -24,7 +23,15 @@ It's a good idea to give visitors an easy way to send such requests; for example
 
 ## Count
 
-Optionally, a GET request to the same URL could respond with the emoji reaction counts.
+Optionally, a GET request to the same URL may respond with the emoji reaction counts.
```
Comment (muan) [inline README.md:26]:
```suggestion
Optionally, a `GET` request to the same URL may respond with the emoji reaction counts.
```

File: README.md (hunk 1 above)
Comment (muan) [reply README.md:34]: 403 probably? I'll update.

File: README.md [hunk 6]
```diff
@@ -1,16 +1,24 @@
 # Open Heart Protocol
 
-Open Heart Protocol lets an anonymous user sends an emoji reaction to a URL.
+The Open Heart protocol lets an anonymous user sends an emoji reaction to a URL.
+
+<div class="reactions">
+<open-heart href="https://api.openheart.fyi" emoji="❤️"></open-heart>
+<open-heart href="https://api.openheart.fyi" emoji="🫀"></open-heart>
+<open-heart href="https://api.openheart.fyi" emoji="🥨"></open-heart>
+</div>
```
Comment (muan) [inline README.md:9]: These will be stripped out on github.com but show on on the GitHub Pages site.

File: README.md (hunk 4 above)
Comment (muan) [reply README.md:18]: Using https://api.openheart.fyi for the example now.

**Review by muan — APPROVED** [review]

**Pull request merged**